src/d3a/d3a_core/sim_results/endpoint_buffer.py

```diff
@@ -77,7 +78,7 @@ def generate_json_report(self):
             "cumulative_loads": self.cumulative_loads,
             "price_energy_day": self.price_energy_day,
             "cumulative_grid_trades": self.cumulative_grid_trades,
-            "bills": self.bills,
+            "bills": self.bills_redis,
```

Comment:
Shouldn't this change be reverted to export self.bills locally?

Reply:
Oops, correct. I will revert this, thanks